suite-common/message-system/src/messageSystemUtils.ts

```diff
@@ -327,3 +330,121 @@ export const resolveMessageContent = (localizedMessages: Localization, language:
 };
+
+export const toMessageSystemOptions = <T extends string>(
+    values: readonly T[],
+): { label: string; value: T }[] =>
+    values.map(value => ({
+        value,
+        label: value.replace(/^./, char => char.toUpperCase()),
+    }));
+
+export const getDefaultActionByCategory = (category: Category): Action => {
+    const defaultLocalization = {
+        en: '',
+        cs: '',
+        es: '',
+        de: '',
+        fr: '',
+        pt: '',
+    };
+
+    const baseMessage = {
+        id: uuidv4(),
+        priority: 100,
+        dismissible: true,
+        variant: 'info' as const,
+        category,
+        content: defaultLocalization,
+    };
+
+    const extraFields = (() => {
+        switch (category) {
+            case 'modal':
+                return {
+                    modal: {
+                        title: defaultLocalization,
+                        image: '',
+                    },
+                };
+            case 'context':
+                return {
+                    context: {
+                        domain: '',
+                    },
+                };
+            case 'feature':
+                return {
+                    feature: [
+                        {
+                            domain: '',
+                            flag: true,
+                        },
+                    ],
+                };
+            case 'banner':
+            default:
+                return {};
+        }
+    })();
+
+    return {
+        message: {
+            ...baseMessage,
+            ...extraFields,
+        },
+        conditions: [],
+    };
+};
+
+export const getDefaultConditionValue = (key: string) => {
+    switch (key) {
+        case 'duration': {
+            const now = new Date();
+            const from = new Date(now.setHours(0, 0, 0, 0)).toISOString();
+            const to = new Date(now.setFullYear(now.getFullYear() + 1)).toISOString();
+
+            return { from, to };
+        }
+
+        case 'environment':
+            return { desktop: '*', mobile: '*', web: '*' };
+
+        case 'os':
+            return {
+                macos: '*',
+                linux: '*',
+                windows: '*',
+                android: '*',
+                ios: '*',
+                chromeos: '*',
+            };
+
+        case 'browser':
+            return { firefox: '*', chrome: '*', chromium: '*' };
+
+        case 'transport':
+            return { bridge: '*', webusbplugin: '*' };
+
+        case 'settings':
+            return [{ tor: false }];
+
+        case 'devices':
+            return [
+                {
+                    model: 'T',
+                    firmwareRevision: '',
+                    firmware: '*',
+                    bootloader: '*',
+                    variant: '*',
+                    vendor: 'trezor.io',
+                },
+            ];
+
+        case 'countryCodes':
+            return ['US'];
```

Comment:
```suggestion
            return ['CZ'];
```

so that it works without change on our VPN